chain/bitcoind_client.go

```diff
@@ -139,30 +146,77 @@ func (c *BitcoindClient) GetBlockHeight(hash *chainhash.Hash) (int32, error) {
 	return header.Height, nil
 }
 
+// GetBlockAsync returns an instance of a type that can be used to get the
+// result of the RPC at some future time by invoking the Receive function on the
+// returned instance.
+func (c *BitcoindClient) GetBlockAsync(
```

Comment:
If all of these `*Async` methods return a `Future*` result and never an error, do we even need the error return value? Might make usage easier to not have to check for errors?